package.json

```diff
@@ -53,6 +53,7 @@
     "dangerously-set-inner-html": "^2.0.1",
     "exports-loader": "^0.7.0",
     "inputmask": "^3.3.11",
+    "jquery": "^3.3.1",
```

Comment:
@New0 Make sure that jQuery is not being bundled (by webapack) should be handled here https://github.com/CalderaWP/Caldera-Forms/blob/master/webpack.config.js#L52) 

Also, probably should use 1.12.4 since that's what WordPress uses.